README.md

```diff
@@ -1,18 +1,16 @@
 # shared-workflows
 
-A public-facing, centralized place to store reusable GitHub workflows and action
-used by Grafana Labs. See the `actions/` directory for the individual actions
-themselves.
+A public-facing, centralized place to store reusable workflows and GitHub Actions used by Grafana Labs.
```

Comment:
The word "public-facing" on this repository's README has always grated on me. I wonder if we can come up with a better way to phrase this sentence.